hack/tools/tilt-prepare/main.go

```diff
@@ -570,6 +582,87 @@ func certManagerTask() taskFunction {
 	}
 }
 
+// kustomizationFile represents a kustomization.yaml.
+type kustomizationFile struct {
```

Comment:
We already import `sigs.k8s.io/kustomize/api` in the main `go.mod` file (not in `hack/tools/go.mod` yet). If it is already in the main one, why not reusing the structs from there for marshaling?!

Reply:
Good point. I'll take a look. I just assumed that would pull in to much stuff. But if they have a clean api package and we're already using it - why not

Reply:
(We get it transitively by importing kubectl for clusterctl)

its in `sigs.k8s.io/kustomize/api/types`